flask_rebar/utils/marshmallow_objects_helpers.py

```diff
@@ -0,0 +1,28 @@
+try:
+    import marshmallow_objects as mo
+
+    MARSHMALLOW_OBJECTS = True
+except ImportError:
+    MARSHMALLOW_OBJECTS = False
+
+
+def get_marshmallow_objects_schema(model):
+    if MARSHMALLOW_OBJECTS and (
+        isinstance(model, mo.Model) or issubclass(model, mo.Model)
+    ):
+        return model.__get_schema_class__()
+    else:
+        return None
+
+
+if MARSHMALLOW_OBJECTS:
```

Comment:
I'm thinking though how this will behave if MARSHMALLOW_OBJECTS is False and I think it might be confusing.

If I try "from marshmallow_objects_helpers import NestedTitledModel" and marshmallow_objects is not installed, then I'll get an import error along the line of 'can't import name "NestedTitleModel" from marshmallow_objects_helpers'. 

It might be better to have NestedTitleModel defined, but have it throw an error on instantiation?

Reply:
I like that - I considered just creating a NOOP class but thought "eeh I don't want it to fail quietly" - will add something that fails loudly with a clear error message